server/migrations/20210914210024-clean-up-db.js

```diff
@@ -8,7 +8,14 @@ module.exports = {
         'SELECT id FROM "Addresses";',
         { transaction },
       );
-      const addressIds = addresses[0].map((_) => _.id).join(', ');
+
+      let addressIds = addresses[0].map((_) => _.id).join(', ');
```

Comment:
Any reason we're editing an old migration rather than making a new one?

Reply:
Because this is not actually changing anything in the database. This is just a fix of the migration - this migration caused an error if you ran it with an empty Address table.